TestTables/CMIP7_CV.json

```diff
@@ -0,0 +1,297 @@
+                "tpt":"point",
+                "tstat":"statistic",
+                "tsum":"sum"
+            },
+            "vertical_label":{
+                "1000hPa":"1000 hPa",
+                "100hPa":"100 hPa",
+                "10hPa":"10 hPa",
+                "220hPa":"220 hPa",
+                "500hPa":"500 hPa",
+                "560hPa":"560 hPa",
+                "700hPa":"700 hPa",
+                "840hPa":"840 hPa",
+                "850hPa":"850 hPa",
+                "d0m":"surface",
+                "d100m":"100m depth",
+                "d10cm":"1cm depth",
+                "d1m":"1m depth",
+                "d2000m":"2000m depth",
+                "d300m":"300m depth",
+                "d700m":"700m depth",
+                "h100m":"100m height",
+                "h10m":"10m height",
+                "h16":"16 height levels",
+                "h2m":"2m height",
+                "h40":"40 height levels",
+                "l":"model level",
+                "p19":"19 pressure levels",
+                "p27":"27 pressure levels",
+                "p3":"3 pressure levels",
+                "p39":"39 pressure levels",
+                "p4":"4 pressure levels",
+                "p7c":"7 pressure levels",
+                "p7h":"7 pressure levels",
+                "p8":"8 pressure levels",
+                "rho":"density surface",
+                "u":"unspecified (no vertical dimension)"
+            }
+        },
+        "data_specs_version":"CMIP-7.0.0.0",
+        "experiment_id":{
+            "1pctCO2":{
+                "activity_id":[
+                    "CMIP"
+                ],
+                "additional_allowed_model_components":[
+                    "AER",
+                    "CHEM",
+                    "BGC"
+                ],
+                "description":"DECK: 1pctCO2",
+                "end_year":"",
+                "experiment":"1 percent per year increase in CO2",
+                "experiment_id":"1pctCO2",
+                "min_number_yrs_per_sim":"150",
+                "parent_activity_id":[
+                    "CMIP"
+                ],
+                "parent_experiment_id":[
+                    "piControl"
+                ],
+                "required_model_components":[
+                    "AOGCM"
+                ],
+                "start_year":"",
+                "sub_experiment_id":[
+                    "none"
+                ],
+                "tier":"1"
+            }
+        },
+        "frequency":{
+            "1hr":{
+                "approx_interval":0.041666666666666664,
+                "description":"sampled hourly"
+            },
+            "1hrCM":{
+                "approx_interval":0.041666666666666664,
+                "description":"monthly-mean diurnal cycle resolving each day into 1-hour means"
+            },
+            "3hr":{
+                "approx_interval":0.125,
+                "description":"3 hourly mean samples"
```

Comment:
There will be some time: mean and some time: point variables sharing this frequency. Would it be best to replace this with just "3 hourly"?

Reply:
Yes, definitely.